cmd/afsPmf/afsPmf.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/popgen"
+	"log"
+)
+
+func afsPmf(vcfFile string, outFile string, unPolarized bool) {
+	g, err := popgen.VcfToAfs(vcfFile, !unPolarized) //VcfToAfs is written in terms of polarized, so this is inverted here.
+	exception.FatalOnErr(err)
+	out := fileio.EasyCreate(outFile)
+
+	numAlleles := g.Sites[0].N
+	frequencySlice := make([]int, numAlleles)
+
+	for _, site := range g.Sites {
+		frequencySlice[site.I]++
+	}
+	_, err = fmt.Fprintf(out, "AlleleFrequency\tNumAlleles\n")
```

Comment:
I am thinking this is a header labeling two columns, but it seems like our print statement below outputs three columns

Reply:
fixed in the next commit